{"name": "test_get_price_fail", "status": "failed", "statusDetails": {"message": "AssertionError: assert 4 == 3", "trace": "self = <test_price1.TestTokenPrice object at 0x7feec00e30d0>\n\n    @allure.feature('币种信息错误')\n    @allure.story('获取不到币种信息')\n    def test_get_price_fail(self):\n>       assert 4 == 3\nE       assert 4 == 3\n\ncases/test_price1.py:29: AssertionError"}, "start": 1620843372916, "stop": 1620843372916, "uuid": "b9693fc5-bf19-4e90-887f-ac2854ad8ba8", "historyId": "d5479ec6e0edff03c633b4658de309e9", "testCaseId": "785e095beda56a398fe1fb20c9a7381b", "fullName": "cases.test_price1.TestTokenPrice#test_get_price_fail", "labels": [{"name": "feature", "value": "api_test"}, {"name": "story", "value": "验证获取币种价格"}, {"name": "feature", "value": "币种信息错误"}, {"name": "story", "value": "获取不到币种信息"}, {"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test_price1"}, {"name": "subSuite", "value": "TestTokenPrice"}, {"name": "host", "value": "192.168.0.102"}, {"name": "thread", "value": "47553-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test_price1"}]}